### ISSUE TITLE
Maybe duplication of code? bits() vs bitsize()

### ISSUE THREAD

**Issue by Harding-Stardust** [arizvisa/ida-minsc#171]
https://github.com/arizvisa/ida-minsc/blob/af07f167a0992aed3f3a2a1670593880baa57820/base/database.py#L379C19
vs
https://github.com/arizvisa/ida-minsc/blob/af07f167a0992aed3f3a2a1670593880baa57820/base/database.py#L404C7-L404C7

Can these be different?

**Comment by arizvisa** [comment]
Yes. One of them is returning the size of a pointer type for the database (which uses the compiler and its type library), and the other is returning the number of bits used by the database itself. Your disassembler uses this to distinguish the length of an address and internally it represents which version if `idaapi.BADADDR` gets used by the disassembler api.

Closing. Hopefully this answered your question. If it hasn't, let me know and I can re-open it if necessary.

**Comment by arizvisa** [comment]
Got it. I'll consider renaming it to `pointer_size` in that case. In the next release the `config` namespace will actually end up being renamed to `information` as it doesn't actually have a real purpose. Moreso, it's really just returning information that the disassembler explicitly stores for a database. Things like the author name, original filename, etc. Crap like that anyways.
